docs/userGuide/siteJsonFile.md

```diff
@@ -156,6 +159,11 @@ The following properties will apply to `index.md`:
 </box>
 </span>
 
+#### **`pagesExclude`**
+**An array of file patterns to be excluded from rendering.** The exclusion pattern follows the glob syntax.
+
+This property is the global variant to the `globExclude` property and is functionally identical to it. If both properties are used simultaneously, the file patterns will be combined into one.
```

Comment:
would `... combined into one.` -> `The file patterns from both properties will be combined when excluding pages` be clearer?

Reply:
Yes I feel that would make the sentence clearer and concise. I'll reword that part.